src/Ledger/Utxo/Properties.lagda

```diff
@@ -39,23 +40,35 @@ open Tx
 open Equivalence
 open Properties
 
-import Data.Nat.Tactic.RingSolver as ℕ
 open Tactic.EquationalReasoning.≡-Reasoning {A = ℕ} (solve-macro (quoteTerm +-0-monoid))
 
 instance
   _ = TokenAlgebra.Value-CommutativeMonoid tokenAlgebra
   _ = +-0-monoid
 
+record Add (A : Set) : Set where
+  infixl 7 _+_
+  field _+_ : A → A → A
+
+open Add ⦃ ... ⦄ public
+
+instance
+  addInt : Add ℤ
+  addInt = record { _+_ = _+ℤ_ }
+
+  addNat : Add ℕ
+  addNat = record { _+_ = _+ℕ_ }
+
+  addCommMon : Add (TokenAlgebra.Value tokenAlgebra)
+  addCommMon = record { _+_ = _+ᵛ_ }
+
 private variable
-  tx : TxBody
-  utxo utxo' utxo1 utxo2 : UTxO
-  fee fee' fees fees' : Coin
-  utxoState utxoState' utxoState1 utxoState2 : UTxOState
-  deposits deposits' : DepositPurpose ⇀ Coin
-  donation donations donations' : Coin
-  Γ : UTxOEnv
-  s s' : UTxOState
-  A : Set
+  tx                               : TxBody
+  utxo utxo'                       : UTxO
+  Γ                                : UTxOEnv
+  utxoState utxoState'             : UTxOState
+  fees fees' donations donations'  : Coin
+  deposits deposits'               : DepositPurpose ⇀ Coin
```

Comment:
Some unused private variables were eliminated.  Alignment is merely to make it easier for me to read and refer to. (I do realize this part of the code will not make it into the documentaion.)

Reply:
reverted